### PR TITLE
Fixes #13299:  add urlencode filter.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/13299

### CODE REVIEW

File: test/utils/urlencode.filter.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+describe('Filter:urlencode', function() {
+    var scope = {};
```
Comment (ehelms) [inline test/utils/urlencode.filter.test.js]: Technically this scope isn't even needed to be attached to correct? I'm thinking that if someone looked at this for how to use it, they might interpret it that way vs just using the filter/service itself.

Comment (waldenraines) [inline test/utils/urlencode.filter.test.js]: Yeah, this was a bad copy/paste from another test.  Will fix.

**Comment by waldenraines** [comment]
@ehelms updated.

**Comment by ehelms** [comment]
ACK

**Pull request merged**